src/cf-tables/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-tables",
-  "version": "1.1.0",
```

Comment:
I might want to double check with @contolini, but I don't think there's any need to update the version. The bot will do it according to what's in the changelog.